### ISSUE TITLE
unable to check chat requirement

### ISSUE THREAD

**Issue by lianguooo** [xqdoo00o/ChatGPT-to-API#64]
post: /v1/chat/completions
data:
{
    "model": "gpt-3.5-turbo",
    "messages": [
        {
            "role": "system",
            "content": "You are a helpful assistant."
        },
        {
            "role": "user",
            "content": "Hello!"
        }
    ]
}
response: {"error":"unable to check chat requirement"}

但是请求模型正常
get: /v1/models
response: 
{
    "data": [
        {
            "created": 1688888888,
            "id": "gpt-3.5-turbo",
            "object": "model",
            "owned_by": "chatgpt-to-api"
        },
        {
            "created": 1688888888,
            "id": "gpt-4",
            "object": "model",
            "owned_by": "chatgpt-to-api"
        }
    ],
    "object": "list"
}

**Comment by xqdoo00o** [comment]
参考[之前issue](https://github.com/xqdoo00o/ChatGPT-to-API/issues/37#issuecomment-2154136023)

**Comment by lianguooo** [comment]
> 参考[之前issue](https://github.com/xqdoo00o/ChatGPT-to-API/issues/37#issuecomment-2154136023)

作者您好，我环境变量配置了，请求仍然异常：unable to check chat requirement，har 文件也已更新。
以下是我的环境变量：
export CLIENT_PROFILE=Chrome_125
export UA="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36"

**Comment by xqdoo00o** [comment]
ip呢，是否可用浏览器正常登录使用

**Comment by lianguooo** [comment]
IP 是可用的，profile设置为chrome_125也是同样的问题，是不是125版本太高了。

**Comment by lianguooo** [comment]
> > IP 是可用的，profile设置为chrome_125也是同样的问题，是不是125版本太高了。
> 
> 键值哪里有125的版本。。。？ 还有ua不要双引号

您好，因为我更新为firefox_123了，但仍无法使用。UA不加双引号会无法启用，因为包含空格了，麻烦了

**Comment by lianguooo** [comment]
> > IP 是可用的，profile设置为chrome_125也是同样的问题，是不是125版本太高了。
> 
> 键值哪里有125的版本。。。？ 还有ua不要双引号

您好，我现更新为firefox_123了，但仍无法使用。UA不加双引号会无法启用，因为包含空格了，麻烦了

**Comment by lianguooo** [comment]
> 哦，你是直接export，不是docker-compose.yml。你直接官网不出盾？你服务器ipv4 ipv6都有？
直接访问官网会有人机验证，但是勾选后可进入网站，是这个原因造成的吗？

**Comment by bi1101** [comment]
Hi sir, I have added environment variables as follows in docker but the issue persists. 

The Ip is from my PC and I can access chatgpt.com normally, no CAPTCHA required. I also noticed on the official websites, the chat requirement sometimes return 403 error codes but the UI seems to auto retry until it gets 200, could this be the reason for this error?


CLIENT_PROFILE = chrome_124
UA = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36 Edg/125.0.0.0
